components/select/Select.razor.cs

```diff
@@ -720,6 +720,16 @@ protected override void OnInitialized()
             }
             _isInitialized = true;
 
+            if (_defaultValueIsNotNull)
+            {
+                Value = DefaultValue;
+            }
+
+            if (_defaultValuesHasItems)
```

Comment:
This seems to me it could be an OR (`||`) with previous condition.

Reply:
Do you mean

```C#
if (_defaultValueIsNotNull || _defaultValuesHasItems) {
    Value = DefaultValue;
    Values = DefaultValues;
}
```